libs/platform/src/lib/components/form/checkbox-group/checkbox-group.component.ts

```diff
@@ -54,6 +54,18 @@ export class CheckboxGroupComponent extends CollectionBaseInput {
     @Input()
     isInline = false;
 
+    /**
+     * Establishes two way binding, when checkbox group used outside form.
+     */
+    @Input()
```

Comment:
Why we can not use valueChanged ?

Reply:
In valueChanged, we have passed an object which has a property which denotes which checkbox was changed and the final values of checked checkboxes. 

Here `value` is not `Input` for two way binding we need only checked checkboxes value. 
So for avoiding any breaking change, we need to introduce `checkedChange` property.